recipes/qxmpp/all/conanfile.py

```diff
@@ -27,62 +33,90 @@ class QxmppConan(ConanFile):
         "with_gstreamer": False,
```

Comment:
```suggestion
        "with_gstreamer": False,
        "with_qt": 5,
```

Please, add an option for Qt, the upstream supports both version 5 and 6.

```
with_qt: [5, 6]
```


Using version 5 as default will build this package and make it available in Conan Center. Some users may be affect in case using Qt6, but who is using Qt6 right now, needs to build from source, so they probably are using a profile or something prepared.